src/plugins/telemetry/common/routes.ts

```diff
@@ -10,9 +10,3 @@
  * Fetch Telemetry Config
  */
 export const FetchTelemetryConfigRoute = '/api/telemetry/v2/config';
```

Comment:
This is kind of an interesting one... I am deferring changing this path as I am not familiar with the original context for introducing `v2` in it, but I am guessing once an actual versioning mechanism is available we can come back to it? Let me know what y'all think.

Reply:
That is a very good question!! I need to rely on @Bamieh because the `/v2/` part of the telemetry APIs predates me 🤷 

FWIW, our telemetry APIs should be considered internal (we probably should name them as such, TBH)... Having said that, we are aware that some external products already use them, so I don't think we can introduce a breaking change without notifying them (or providing a sensible deprecation period).

Reply:
> the /v2/ part of the telemetry APIs

From what I can recall, 'v2' deals with encryption

Reply:
The versioning is introduced for external consumers to check for and detect how to handle incoming requests (our telemetry endpoint for example).

v1 -> v2 was for encryption. This way the telemetry cluster endpoint will check the request URL and will handle the request differently based on the version.